src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -4111,7 +4111,7 @@ private ExternalScriptInfo FindLocalizedModuleManifest(string path)
 
                 String filePath = stringBuilder.ToString();
 
-                if (Utils.NativeFileExists(filePath))
+                if (File.Exists(filePath))
```

Comment:
Why not use `Utils.FileExists` ?

Reply:
Utils.FileExists behavior is the same (no exception is thrown). So it is better to use the standard method.

Reply:
It's better to make all the `Utils.NativeFileExists -> File/Directory.Exists` changes in a separate PR. So for now, just do the simple replacement of `Utils.NativeFileExists -> Utils.FileExists`.